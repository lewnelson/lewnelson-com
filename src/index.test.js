import ReactDOM from 'react-dom'
require('./index')

test('it renders the app to the DOM', () => {
  expect(ReactDOM.render.mock.calls[0]).toMatchSnapshot()
})
