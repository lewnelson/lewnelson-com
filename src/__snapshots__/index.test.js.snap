// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the app to the DOM 1`] = `
Array [
  <App />,
  "app",
]
`;
